### DISCUSSION TITLE
Tutorial Request: Modelling Units and Measurements and the use of UoM

### DISCUSSION THREAD

**Discussion by shawntanzk** [jamesaoverton/obook#413] [Tutorial requests]
**Tutorial Request:**

How to model units and measurements in OBO world (or perhaps how are units and measurements modelled in OBO world) - UO or QUDT? how to use UoM to handle modelling differences? Practical examples of those 

**Suggested materials**

https://qqv.ontodev.com/primer.html
https://github.com/pato-ontology/pato/issues/101
https://units-of-measurement.org/

**Comment by matentzn** [comment]
Excellent suggestion @shawntanzk.

I think @jamesaoverton would be perfect for this, I will see if I can convince him for a late June tutorial!

**Comment by shawntanzk** [comment]
Thanks heaps :) really appreciate it!

**Comment by StroemPhi** [comment]
I also seek pragmatic solutions to this issue and just wanted to point out how Allotrope is modeling it in AFO. They introduce an ICE called facet of which [quality quantification facet](https://www.ebi.ac.uk/ols4/ontologies/afo/classes/http%253A%252F%252Fpurl.allotrope.org%252Fontologies%252Fresult%2523AFR_0001583?lang=en) is a specialization used to provide the value of a quality. Although this AFO approach entails what Chris calls a shadow hierarchy, I think it is better to speak of the values as kinds of ICEs as they are not the attributes themself but the assertions/descriptions we make after having observed/percieved these attributes.

**Comment by matentzn** [comment]
This has been delivered by @jamesaoverton  https://oboacademy.github.io/obook/tutorial/units/
